src/VisualStudio/Core/Def/CodeCleanup/AbstractCodeCleanUpFixer.cs

```diff
@@ -140,39 +130,28 @@ private async Task<bool> FixHierarchyContentAsync(IVsHierarchyCodeCleanupScope h
     }
 
     private Task<bool> FixSolutionAsync(Solution solution, ICodeCleanUpExecutionContext context)
-    {
-        return FixAsync(_workspace, ApplyFixAsync, context);
-
-        // Local function
-        Task<Solution> ApplyFixAsync(IProgress<CodeAnalysisProgress> progress, CancellationToken cancellationToken)
-        {
-            return FixSolutionAsync(solution, context.EnabledFixIds, progress, cancellationToken);
-        }
-    }
+        => FixAsync(
+            solution.Workspace,
+            (progress, cancellationToken) => FixProjectsAsync(
+                _globalOptions, solution, solution.Projects.Where(p => p.SupportsCompilation).ToImmutableArray(), context.EnabledFixIds, progress, cancellationToken),
+            context);
```

Comment:
just inlined the local function version.  Note: FixSolution became FixProjects where the set of projects to fix is passed in.